lib/AST/Decl.cpp

```diff
@@ -5562,6 +5562,30 @@ AccessorDecl *AccessorDecl::create(ASTContext &ctx,
   return D;
 }
 
+bool AccessorDecl::isAssumedNonMutating() const {
+  switch (getAccessorKind()) {
+    case AccessorKind::Get:
+    case AccessorKind::Address:
+    case AccessorKind::Read:
+      return true;
+
+    case AccessorKind::Set:
+    case AccessorKind::WillSet:
+    case AccessorKind::DidSet:
+    case AccessorKind::MutableAddress:
+    case AccessorKind::Modify:
+      return false;
+  }
+  llvm_unreachable("bad addressor kind");
```

Comment:
Copy/paste error:"addressor" instead of "accessor". Also, the switch statement is over-indented.